<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CRegistryFileProgressBaseDialog</class>
 <widget class="QDialog" name="CRegistryFileProgressBaseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>289</width>
    <height>173</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Registry File Progress</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="textLabel9">
       <property name="text">
        <string>Reading registry file</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_editregistryfile"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="m_editextensionstring"/>
   </item>
   <item>
    <widget class="QProgressBar" name="m_progressbar"/>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <spacer name="spacer2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>80</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_pushbuttoncancel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>70</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>QProgressBar</class>
   <extends>QFrame</extends>
   <header>QProgressBar</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
